packages/eslint-plugin-kolibri/lib/rules/vue-no-unused-translations.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @fileoverview Disallow unused translation string definitions.
+ */
+
+'use strict';
+
+const eslintPluginVueUtils = require('eslint-plugin-vue/lib/utils');
+
+const utils = require('../utils');
+const constants = require('../constants');
+
+const { GROUP_$TRS } = constants;
+
+const $TR_FUNCTIONS = ['$tr', 'coachString', 'coreString', 'learnString'];
```

Comment:
Code looks good!

I only have one note for the new version - I would suggest moving translations functions configuration rather to the rule settings. We might need to override it from other projects. So far I only know about Studio and I am actually not 100% sure if these rules are turned on there, but it would make sense to me anyway since this is project specific unlike other logic. I guess that it might be a bit more intuitive for Kolibri folks to just update eslintrc, too.

Reply:
I concur - our linting tools are currently used on Studio, and while this could be turned off, I think it would be a useful rule to leverage there.

Reply:
In the kolibri `.eslintrc.js` we could then just directly import the namespaces from the `ProfileStrings.js` file!